libraries/botbuilder-ai/src/numberWithUnits.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+*/
+
+/** A number together with units returned by a recognizer. */
+export interface NumberWithUnits {
+    /** A recognized number if any. */
+    readonly number?: number;
```

Comment:
>number [](start = 13, length = 6)

Should this be value or something? Number isn't a reserved word but it's a type in TypeScript